### PR TITLE
Revamp conversation view / message view styles

### PR DESCRIPTION
**Pull request by m4dz**

### CODE REVIEW

File: client/app/styles/_conversation.styl [hunk 1]
```diff
@@ -42,227 +25,347 @@
...
-            .hour
-                margin-left auto
-                color darkcolor
-                white-space pre
+    // .thread
+    //     padding 0
 
-            &:hover:not(.active)
-                background #f4f4f4
+    //     .message
```
Comment (frankrousseau) [inline client/app/styles/_conversation.styl]: dead code should be removed.

**Comment by m4dz** [comment]
> dead code should be removed.

Oh yes, I've a huge "big code cleaning" commit next :wink:.

**Comment by frankrousseau** [comment]
Can you push it?

**Comment by aenario** [comment]
@frankrousseau , @m4dz only said it on the #emailsprint channel, but this PR is still WIP

**Comment by m4dz** [comment]
Add it still have the label :D

**Comment by frankrousseau** [comment]
Ok sorry.

**Comment by m4dz** [comment]
No problem : we're all quite under water, and this refactoring / restyling is quite long :confused:. Thanks for your patience…

**Comment by m4dz** [comment]
All right! Thanks to all for your patience, this PR in now ready to be reviewed (last rebase on 8074c5c).

File: client/app/components/message_header.coffee [hunk 2]
```diff
@@ -0,0 +1,118 @@
+{div, span, table, tbody, tr, td, img, a, i} = React.DOM
+MessageUtils = require '../utils/message_utils'
+
+ContactStore   = require '../stores/contact_store'
+
+{MessageFlags} = require '../constants/app_constants'
+
+
+(Array.prototype.last = -> this[this.length - 1]) unless Array.prototype.last
```
Comment (aenario) [inline client/app/components/message_header.coffee]: use lodash.last or inline (only one use)

Comment (m4dz) [inline client/app/components/message_header.coffee]: OOohhhh my bad :cry:… Thanks a lot! Of course, `_.last` is **exactly** what I expected…

**Pull request merged**